src/common/generator.cpp

```diff
@@ -0,0 +1,32 @@
+#include "generator.h"
+
+namespace faker::common
+{
+
+thread_local bool is_seeded = false;
+thread_local std::mt19937_64 generator;
+
+std::mt19937_64& GetGenerator()
```

Comment:
it can be directly in `include` directory and we can drop `common` namespace, just `faker`